packages/embarkjs-ens/.babelrc.js

```diff
@@ -0,0 +1,46 @@
+/* global module require */
```

Comment:
Do you really need this file?

Reply:
`embarkjs-ipfs|swarm|whisper` have it too. It was carried over from `packages/embarkjs` (where it is needed) but given how those and this `embarkjs-*` module are setup I'm not 100% sure it's necessary.